R/ExperimentHub-class.R

```diff
@@ -92,8 +92,17 @@ setMethod("cache", "ExperimentHub",
     ## Try to install/load; don't fail if not available
     success <- paste0("see ?", pkg, " and browseVignettes('",
                       pkg, "') for documentation")
-    if (!pkg %in% rownames(installed.packages()))
-        BiocManager::install(pkg, suppressUpdates=TRUE)
+    nopkg <- paste0(pkg,
+                    " not installed.\n  Full functionality, documentation, and loading of data might not be possible without installing")
+    if (!pkg %in% rownames(installed.packages())){
```

Comment:
Minor: I'd use the `paste()` as an opportunity to format the message more readably, particularly respecting the an 80-column limit on text
```r
    nopkg <- paste0(
        "'", pkg, "' not installed.\n  Full functionality, documentation, "
        "and loading of data might not be possible without installing"
    )
```